orcestra_book/attribute_convention.md

```diff
@@ -7,25 +7,39 @@ The goal of this convention is to define a minimal set of global attributes for
 * Automatically retrieve contact information for any dataset
 * Create a summary table with all datasets and some key metadata
 
+For additional metadata not covered by this convention we recommend following the [CF](https://cfconventions.org) and [ACDD](https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3) conventions.
+
 ## Global attributes
 
+The following section lists a selection of required and recommended global attributes for datasets.
+The required attributes are essential for the [ORCESTRA Data Browser](https://browser.orcestra-campaign.org) to render proper landing pages.
+
+If an attribute is undefined, it is recommended not to set it instead of using empty strings.
+
+### Required
+
 Key | Value
 --- | ---
 `title` | Short phrase or sentence describing the dataset
 `summary` | Paragraph describing the dataset
 `creator_name` | Comma-separated list of names
 `creator_email` | Comma-separated list of emails
+`license` | [SPDX-ID](https://spdx.org/licenses/)
+
+### Recommended
+
+Key | Value
+--- | ---
 `project` | Comma-separated list of projects (see below)
 `platform` | Name of the platform that supported the sensor (see below)
 `source` | Method of production of the original data (e.g. "radar", "radiometer", "CTD")
 `history` | Audit trail for modifications to the original data
-`license` | [SPDX-ID](https://spdx.org/licenses/)
 `references` | Comma-separated list of URL/DOI to extended information <br/>Published or web-based references that describe the data or methods used to produce it.
 `keywords` | Comma-separated list of keywords
-
-Undefined attributes **should not** be set.
-
-For additional metadata we encourage to follow [CF](https://cfconventions.org) and [ACDD](https://wiki.esipfed.org/Attribute_Convention_for_Data_Discovery_1-3) conventions
+`processing_level` | Processing level of the dataset (cf. eg. [NASA](https://www.earthdata.nasa.gov/learn/earth-observation-data-basics/data-processing-levels))
```

Comment:
ACDD defines this as a textual description, not as e.g. a number from the NASA processing levels. I think we should find a wording which doesn't suggest we have to adhere to the NASA definition. (For some non-satellite instruments, it can lead to a very odd processing chain, when trying to follow their definition)

Reply:
I just went with the ACDD definition. If anyone will set this at all, it is more likely that they can explain what was done in a textual way (or by referencing a script)